.gitignore

```diff
@@ -3,5 +3,6 @@ Packages
 .swiftpm
 xcuserdata
 *.xcodeproj
+!WallEView.xcodeproj
```

Comment:
Had to add it so that we don't have to recreate project every time. Not sure if We already can use SPM for application targets?

Reply:
I'm a bit lost with the state of SPM those days, but indeed I'm not sure it's yet possible, given that `.app` bundle contain resources (like at least the plist and the app icon) and those are still not yet supported by SPM (though that'll likely change soon)

Reply:
Ok just got confirmation from another Slack, SPM still can't generate macOS bundles so we still need the `xcodeproj` indeed.